contracts/Versus.cdc

```diff
@@ -19,12 +20,11 @@ import Content from "./Content.cdc"
 pub contract Versus {
 
     //A set of capability and storage paths used in this contract
-    pub let VersusAdministratorPrivatePath: PrivatePath
-    pub let VersusAdministratorStoragePath: StoragePath
     pub let VersusAdminClientPublicPath: PublicPath
     pub let VersusAdminClientStoragePath: StoragePath
```

Comment:
```suggestion
    pub let AdminPublicPath: PublicPath
    pub let AdminStoragePath: StoragePath
```

As a suggestion, I would remove `Client` from the name here -- I noticed that the `VersusAdmin` resource is now being stored at `VersusAdminClientStoragePath`, so the naming didn't quite match up.

Also personally I would just say `Admin` instead of `VersusAdmin` to avoid stutter with the contract name (i.e. `Versus.VersusAdmin`).

TBH I got confused by the naming of these types. What if you called `VersusAdminClient` just `AdminPublic`, do be explicit about the public nature of the interface?

So you could have:
- `Versus.Admin` -- performs all admin operations
- `Versus.AdminPublic` -- exposes public function to deposit collection capability

Reply:
Love the suggestion!